en-US/mozorg/internet-health/privacy-security.lang

```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
 Source: <a href="%(pew)s" rel="external">Pew Research Center</a>, May 2015
 
 
+# This is used as a decorative element in a callout box.
+;91%
+91%
+
+
 ;What you can do about user control
 What you can do about user control
 
 
+##TAG: internet_health_subpage_rebrand
+;Control your apps
+Control your apps
+
+
 ;Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 
 
+##TAG: internet_health_subpage_rebrand
+;Manage your preferences
+Manage your preferences
+
+
+;Know your settings. You can manage your profile and preferences for <a href="{{ google }}">Google</a>, <a href="{{ yahoo }}">Yahoo!</a> and <a href="{{ facebook }}">Facebook</a> ads.
```

Comment:
You’ll need to replace variables with `%(google)s`, `%(privacy)s` and `%(facebook)s`, that’s what bedrock will expect